packages/flutter/lib/src/cupertino/constants.dart

```diff
@@ -84,3 +84,9 @@ const Map<CupertinoButtonSize, double> kCupertinoButtonMinSize = <CupertinoButto
   CupertinoButtonSize.medium: 32,
   CupertinoButtonSize.large: 44,
 };
+
+/// The distance a button needs to be moved after being pressed for its opacity to change.
+///
+/// The opacity changes when the position moved is this distance away from the button.
+/// This variable is effective on mobile platforms. For desktop platforms, a distance of 0 is used.
```

Comment:
Can you include a sentence about where you got this value from? So that we know when to update it in the future. For example: "Eyeballed on a physical iPhone 16 running iOS 18.3.1."